mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -15757,6 +15757,384 @@
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": ""
+										},
+										"url": {
+											"raw": "{{protocol}}://{{url}}:{{okapiport}}/eholdings/resources/{{managed-resourceid}}",
+											"protocol": "{{protocol}}",
+											"host": [
+												"{{url}}"
+											],
+											"port": "{{okapiport}}",
+											"path": [
+												"eholdings",
+												"resources",
+												"{{managed-resourceid}}"
+											]
+										}
+									},
+									"response": []
+								}
+							],
+							"_postman_isSubFolder": true
+						},
+						{
+							"name": "Negative",
+							"item": [
+								{
+									"name": "/resources DELETE previously deleted resource id",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"id": "5418623e-d148-449c-bd8a-7f3602cb3676",
+												"exec": [
+													"pm.test(\"Status is 404\", function () {",
+													"    pm.response.to.have.status(404);",
+													"});",
+													"",
+													"pm.test(\"Response must have a json body\", function () {",
+													"    pm.response.to.be.notFound;",
+													"    pm.response.to.have.jsonBody();",
+													"});",
+													"",
+													"var jsonData = pm.response.json();",
+													"",
+													"pm.test(\"Validate schema\", function () {",
+													"     tv4.addSchema(\"JSONAPI.json\", JSON.parse(pm.variables.get(\"schema_jsonapi_content\")));",
+													"    pm.expect(tv4.validate(jsonData, JSON.parse(pm.environment.get(\"schema_jsonapi_content\")))).to.equal(true, \"Schema validation error: \" + JSON.stringify(tv4.error));",
+													"    pm.expect(tv4.missing.length).to.equal(0, \"Missing schemas: \" + JSON.stringify(tv4.missing));",
+													"});",
+													"",
+													"//verify headers",
+													"pm.test(\"'Content-Type'header is present and has 'application/vnd.api+json' value\", function () {",
+													"    pm.response.to.have.header(\"Content-Type\");",
+													"    pm.expect(pm.response.headers.get(\"Content-Type\")).to.be.equal(\"application/vnd.api+json\");",
+													"});",
+													"",
+													"pm.test(\"'Transfer-Encoding' header is present and has 'chunked' value\", function () {",
+													"    pm.response.to.have.header(\"Transfer-Encoding\");",
+													"    pm.expect(pm.response.headers.get(\"Transfer-Encoding\")).to.be.equal(\"chunked\");",
+													"});",
+													"",
+													"pm.test(\"'X-Okapi-Trace' header is present\", function () {",
+													"    pm.response.to.have.header(\"X-Okapi-Trace\");",
+													"});",
+													"",
+													"pm.test(\"Errors array contains 1 entry\", function () {",
+													"   pm.expect(jsonData.errors.length).to.eql(1);",
+													"   pm.expect(jsonData.errors[0].title).to.equal(\"Title not found\");",
```

Comment:
Is this error message "Title not found" or "Resource not found"?

Reply:
it should be `Resource not found` and placed in the resources folder instead of titles. Fixed this issue